### DISCUSSION TITLE
Miguel Q. - No me sale

### DISCUSSION THREAD

**Discussion by Qubemid3** [cpp-review-dune/introductory-review#11] [Q&A] [answered]
Hola Carlos estuve practicando, genere mi carpeta src, luego genere los archivos CMakeLists.txt y main.cc pero al usar asciinema  quize ejecutar pero al intentarlo no me salia podría porfa ver que hice mal.

Aqui mi grabación 

https://asciinema.org/a/ScV9AYNc3iVOKU11VWDfeEUVr

**Comment by carlosal1015** [comment, marked as answer]
Hola @Qubemid3, el primer mensaje de error fue que no encontró el archivo Makefile.

Para poder completar el tutorial necesitamos por ejemplo esos dos archivos
![image](https://user-images.githubusercontent.com/21283014/154614502-6f2192b3-3fc7-46b8-9cbb-694ace7a68e3.png)
Con los siguientes contenidos

![main](https://user-images.githubusercontent.com/21283014/154615200-6f0a8431-e46d-45dc-adf2-44ed7973d8e2.png)
![CMakeLists](https://user-images.githubusercontent.com/21283014/154615072-1e07c747-5f0d-4212-a2fd-e44097c9d6a5.png)

Luego que tengas esos dos archivos, realizar lo [siguiente](https://asciinema.org/a/psh9EZGwfhlaJCWmdi4joxN69).

Postdata: Para que aparezca el prompt de bash usa `asciinema rec -c bash`.

**Reply by carlosal1015** [reply]
También es necesario instalar [`cmake`](https://archlinux.org/packages/extra/x86_64/cmake) con
```
$ yay -Syu cmake --noconfirm
```

**Reply by Qubemid3** [reply]
Hola Carlos, ejecute los comandos como indicaste y esta vez si me salio gracias. Saludos

**Reply by carlosal1015** [reply]
Excelente @Qubemid3, me da mucho gusto, ya estamos listos para mañana.
